frontend/src/app/problems/add/SplitPane.tsx

```
@@ -49,7 +65,7 @@ const SplitPane = ({ leftWidth = 50, leftChildren, rightChildren }: SplitPanePro
             window.removeEventListener("mousemove", handleMouseMove);
             window.removeEventListener("mouseup", handleMouseUp);
         };
-    }, [isResizing]);
+    }, [isResizing,, handleMouseUp, handleMouseMove]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)